### PR TITLE
Update vmssNetworkInterface.json

### PR DESCRIPTION
**Pull request by RajarajeshwariHS**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2017-03-30"
```
Comment (tjprescott) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json]: What is the reason for this change?

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json]: Ping @RajarajeshwariHS

Comment (RajarajeshwariHS) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json]: This was suggested by @number213 .

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json]: Yes, if we're updating API version in `vmssPublicIpAddress`, we should also update it in `vmssNetworkInterface`. I guess the question is what's the reason for changing API version in VMSS specs? Just to use the latest one?

Comment (RajarajeshwariHS) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json]: We use latest Microsoft.Azure.Management.Network nuget which is of version “19.12.0-preview”. I searched in the source repository of this repo and found out the required API : ListVirtualMachineScaleSetPublicIPAddressesWithHttpMessagesAsync
https://github.com/Azure/azure-sdk-for-net/blob/dc23a39caf37d07f02512540d20c29866ff83759/sdk/network/Microsoft.Azure.Management.Network/src/Generated/PublicIpAddressesOperations.cs

Here the apiVersion is set as “2017-03-30”. We require it to be “2018-10-01”.
So, here https://github.com/Azure/azure-rest-api-specs/pull/6692 I have made the required changes to **vmssPublicIPAddress** and as we have to also update **vmssNetworkInterface**, I have make the changes here.

**Review by anton-evseev — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json (hunk 1 above)
Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json]: @RajarajeshwariHS makes sense.
If you need updated Network SDK asap, we can work on getting you a signed nupkg, because the next NuGet release will be in a month

**Review by tjprescott — APPROVED** [review]

**Comment by number213** [comment]
@tjprescott SDK java times out regularly, I guess this is due to the size of Network's specs

**Comment by tjprescott** [comment]
I don't have the ability to merge this while the CI is failing. @sergey-shandar can you take a look?

**Comment by number213** [comment]
Opened PR with this change into Network release branch: https://github.com/Azure/azure-rest-api-specs/pull/7008
